src/app/submission/objects/submission-objects.effects.ts

```diff
@@ -136,7 +136,7 @@ export class SubmissionObjectEffects {
     ofType(SubmissionObjectActionTypes.SAVE_SUBMISSION_FORM_SUCCESS, SubmissionObjectActionTypes.SAVE_SUBMISSION_SECTION_FORM_SUCCESS),
     withLatestFrom(this.store$),
     map(([action, currentState]: [SaveSubmissionFormSuccessAction | SaveSubmissionSectionFormSuccessAction, any]) => {
-      return this.parseSaveResponse((currentState.submission as SubmissionState).objects[action.payload.submissionId], action.payload.submissionObject, action.payload.submissionId);
+      return this.parseSaveResponse((currentState.submission as SubmissionState).objects[action.payload.submissionId], action.payload.submissionObject, action.payload.submissionId, action.payload.notify);
```

Comment:
Not related with this specific change, but it's in this class, I'm getting this blocking error when accessing the submission form and I can't test this PR:

```
ERROR TypeError: Cannot read property 'forEach' of undefined
    at MapSubscriber.project (0.client.js:1282)
    at MapSubscriber../node_modules/rxjs/_esm5/internal/operators/map.js.MapSubscriber._next (client.js:196768)
    at MapSubscriber../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.next (client.js:190809)
    at FilterSubscriber../node_modules/rxjs/_esm5/internal/operators/filter.js.FilterSubscriber._next (client.js:196213)
    at FilterSubscriber../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.next (client.js:190809)
    at ScannedActionsSubject../node_modules/rxjs/_esm5/internal/Subject.js.Subject.next (client.js:190575)
    at SafeSubscriber._next (client.js:112368)
    at SafeSubscriber../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.__tryOrUnsub (client.js:190950)
    at SafeSubscriber../node_modules/rxjs/_esm5/internal/Subscriber.js.SafeSubscriber.next (client.js:190888)
    at Subscriber../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._next (client.js:190832)
```

It's at line 59 - https://github.com/DSpace/dspace-angular/blob/32f8842af8c2e67ca7076221eef241df4b008b31/src/app/submission/objects/submission-objects.effects.ts#L59